client/components/domains/connect-domain-step/connect-subdomain-step-suggested-records.jsx

```diff
@@ -0,0 +1,158 @@
+import { Button } from '@automattic/components';
```

Comment:
This is basically a copy of `ConnectDomainStepAdvancedRecords` - I want to refactor this to not duplicate all this code, but also wanted to keep this PR simple for now 😅 let's refactor this in the future